libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/ImageUtils.java

```diff
@@ -523,17 +523,20 @@ private static boolean resizeImageAndWriteToStream(Context context,
 
         boolean result = bmpRotated.compress(fmt, quality, outStream);
         bmpRotated.recycle();
+        bmpResized.recycle();
```

Comment:
I just moved this line of code at the bottom of the method since was seeing some issues on older devices.
We can remove those calls, but don't think it's hurt having them there.